RRTStar/rrtstar_wrapper.py

```diff
@@ -0,0 +1,48 @@
+import ctypes
+import numpy as np
+from ctypes import c_double, c_int
+
+cdll = ctypes.CDLL("build/libRRTStar.so")
+_c_double_p = ctypes.POINTER(c_double)
+_apply_rrt_star = cdll.ApplyRRTStar
+_apply_rrt_star.argtypes = (
+    c_double, c_double, c_double, c_double, c_int, c_int, _c_double_p,
+    _c_double_p, _c_double_p, _c_double_p, c_int, _c_double_p, _c_double_p
+)
+_apply_rrt_star.restype = c_int
+
+
+def apply_rrt_star(start, end, step_size, max_iterations, obs):
+    """ Run RRTStar given a start, end and list of obstacles.
+
+    Args:
+        start (np.ndarray): [x, y]
+        end (np.ndarray): [x, y]
+        step_size (float): distance to sample new points
+        max_iterations (int): maximum number of iterations to run
+        obs (np.ndarray): [llx, lly, urx, ury]
```

Comment:
Nit: can you please add a comment to obs? It's not totally obvious that these are obstacles.

Reply:
Yup